### DISCUSSION TITLE
[QUESTION] How can I achieve the samve behavior as go-pg's use_zero for pointer fields?

### DISCUSSION THREAD

**Discussion by Warashi** [uptrace/bun#618] [Q&A] [answered]
How can I achieve the same behavior as go-pg's use_zero for pointer fields?
The behavior is the same as use_zero by default for normal fields, but nil in a pointer field is marshaled to null.
In go-pg, I think use_zero will marshal to zero value, but how can I achieve this with bun?

**Comment by vmihailenco** [comment, marked as answer]
Don't use a pointer? E.g. use `string` instead of `*string`.

**Reply by Warashi** [reply]
I want use pointer fields with custom types which implements sql.Scanner and driver.Valuer.
Custom types is struct and not small size, so I want them pointer fields.

**Reply by Warashi** [reply]
sorry, I misunderstood the behavior of go-pg and my codes.
there is no problem with bun.
